frontend/src/individual_edit.tsx

```diff
@@ -0,0 +1,424 @@
+/**
+ * @file This file contains the IndividualEdit function. This function allows a
+ * user (with the required permissions) to edit an individual given by `id`, or
+ * add a new individual if `herdId` is given.
+ */
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles';
+import { get, update } from '@app/communication';
+import { BodyFat, DateBodyfat, DateWeight, Individual, individualLabel,
+         LimitedIndividual, OptionType, ServerMessage,
+        } from '@app/data_context_global';
+import { useMessageContext } from '@app/message_context';
+import { Button, CircularProgress, TextField,
```

Comment:
```suggestion
import { Button, CircularProgress, TextField, InputAdornment,
```
Need to Import InputAdornment to use Kg in prefix in input?
I am not sure what is best here. Maybe @danr  Nows a better way to do this